Makefile

```diff
@@ -112,7 +126,10 @@ include $(BOLOS_SDK)/Makefile.glyphs
 
 APP_SOURCE_PATH += src
 
-ifneq ($(TARGET_NAME),TARGET_NANOS)
+ifeq ($(TARGET_NAME), TARGET_STAX)
+    SDK_SOURCE_PATH += lib_nbgl/src
+    SDK_SOURCE_PATH += lib_ux_stax
+else ifneq ($(TARGET_NAME), TARGET_NANOS)
     SDK_SOURCE_PATH  += lib_ux
```

Comment:
weird that you have HAVE__UX_FLOW for nanos but not lib_ux, is that ok?

Reply:
Don't know really. I kept the same behavior of the previous `Makefile` on this :shrug: